upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -948,11 +948,17 @@ func resolveImage(ec2Client ec2iface.EC2API, name string) (*ec2.Image, error) {
 	if response == nil || len(response.Images) == 0 {
 		return nil, fmt.Errorf("could not find Image for %q", name)
 	}
-	if len(response.Images) != 1 {
-		return nil, fmt.Errorf("found multiple Images for %q", name)
+
+	latest := 0
```

Comment:
Personally I find it easier to keep a pointer to the newest image, but this works :-)